website/views.py

```diff
@@ -88,6 +89,11 @@ def serialize_node_summary(node, auth, primary=True, show_path=False):
             'api_url': node.api_url,
             'title': node.title,
             'category': node.category,
+            'isPreprint': bool(node.preprint_file and node.preprint_file._id),
```

Comment:
https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/node.py#L445
I think you can just do `preprint_file_id` and not `preprint_file` and `preprint_file._id`. The last one is particularly undesirable as the guid (_id) is not on that table